pkg/scheduler/queue/tenant_queues.go

```diff
@@ -127,84 +122,57 @@ func newQueueBroker(maxTenantQueueSize int, forgetDelay time.Duration) *queueBro
 	}
 }
 
-func (qb *queueBroker) len() int {
-	return len(qb.tenantQueues)
+func (qb *queueBroker) isEmpty() bool {
```

Comment:
converted to isEmpty as we only ever used `len` to check `len() == 0`, and isEmpty can be implemented more cheaply on the tree queue than a node count or total queue item count